### PR TITLE
Project cleanup

### PR DESCRIPTION
**Pull request by m00g3n**
**Description**

Changes proposed in this pull request:

- cleanup project structure
- update dependency: module-manager

### CODE REVIEW

File: kyma.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: operator.kyma-project.io/v1alpha1
```
Comment (kwiatekus) [inline kyma.yaml:1]: isn't this file generated by `hack/gen-kyma.sh` ?
I'm not sure if it needs to be versioned

File: Makefile [hunk 2]
```diff
@@ -3,7 +3,7 @@ MODULE_NAME ?= keda
 # Semantic Module Version used for identifying the build
 MODULE_VERSION ?= 0.0.4
 # Module Registry used for pushing the image
-MODULE_REGISTRY_PORT ?= 50092
+MODULE_REGISTRY_PORT ?= 63368
```
Comment (kwiatekus) [inline Makefile:6]: why the new port?

**Review by kwiatekus — COMMENTED** [review]
running make module-build fails

```
➜  keda-manager git:(project-cleanup) make module-build                                                                                                                                        
X Pushing image to "op-kcp-registry.localhost:8888/unsigned"
Error: unable to push config: failed to do request: Head "http://op-kcp-registry.localhost:8888/v2/unsigned/component-descriptors/kyma.project.io/module/keda/blobs/sha256:669e5b66cb3239fc04826839e16df0851798a1cde68d3ae3fefe282c89476d6a": dial tcp 127.0.0.1:8888: connect: connection refused
```

File: kyma.yaml (hunk 1 above)
Comment (m00g3n) [reply kyma.yaml:1]: yes I will remove it

File: Makefile (hunk 2 above)
Comment (m00g3n) [reply Makefile:6]: As we agreed offline we will set it to `8888`

**Review by kwiatekus — COMMENTED** [review]
after running make targets (`module-build` and `module-image`) the git status is dirty.
```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   operator/config/crd/bases/operator.kyma-project.io_kedas.yaml
	modified:   operator/config/manager/kustomization.yaml

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	default.yaml
	template.yaml
```
I think `default.yaml` and `template.yaml` should be git-ignored. I'm not sure how to tackle the modified files

**Review by kwiatekus — APPROVED** [review]

**Pull request merged**